src/std/text.ab

```diff
@@ -100,6 +100,62 @@ pub fun contains(text: Text, phrase: Text): Bool {
   return result == "1"
 }
 
+/// Checks if an array value is in the text.
+pub fun contains_any(text: Text, terms: [Text]): Bool {
+    for term in terms {
+        if contains(text, term) {
+            return true
+        }
+    }
+
+    return false
+}
+
+/// Checks if all the arrays values are in the string
+pub fun contains_all(text: Text, terms: [Text]): Bool {
+    for term in terms {
+        if not contains(text, term) {
+            return false
+        }
+    }
+
+    return true
+}
+
+/// Search all occurences of a regex pattern in the content.
+///
+/// Function uses `sed`
```

Comment:
The comment does not entirely reflect what the function does. It just matches and returns a bool. The found occurrences are not being returned here, so I'd argue that it's not "searching" for them